index.html

```diff
@@ -5502,7 +5504,101 @@ <h2>Base Context</h2>
   }
 }
 </pre>
+      </section>
+
+      <section class="informative">
+        <h3>Differences between Contexts, Types, and CredentialSchemas</h3>
 
+        <p>
+The verifiable credential and verifiable presentation data models leverage a
+variety of underlying technologies including [JSON-LD] and [JSON-SCHEMA-2018].
+This section will provide a comparison of the <code>@context</code>,
+<code>type</code>, and <code>credentialSchema</code> properties, and cover some
+of the more specific use cases where it is possible to use these features of the
+data model.
+        </p>
+        
+        <p>
+The primary purpose of the <code>type</code> property is to uniquely
+identify the type of the <code>verifiableCredential</code> in which it
+appears i.e. to indicate which set of claims the
+<code>verifiableCredential</code> contains. This property is mandatory and
+the value "VerifiableCredential" must appear in the set of values. Whilst
```

Comment:
Avoid use of normative "MUST" language in a non-normative section.

Reply:
This is a compromise we made between @David-Chadwick and I because he felt it was important to restate the sections above.

Reply:
How about "This property, with the value  "VerifiableCredential" is mandatory."

Reply:
I think @TallTed suggestion down below addressed this so that we keep the original intent of restating this is mandatory without using MUST. The text now reads:

>This property, and the value <code>VerifiableCredential</code> within the set of its values, are mandatory.